lib/command_helper.ex

```diff
@@ -0,0 +1,18 @@
+defmodule CommandHelper do
+  def get_command(exec_path, parms) do
+    command =
+      case Application.get_env(:puppeteer_pdf, :is_windows_command) do
```

Comment:
Instead of using `:is_windows_command` as user defined variable, can you change this to `:os.type()` ? 
https://stackoverflow.com/questions/33461345/how-can-i-get-the-current-operating-system-name-in-elixir

Reply:
Great call.  100% cleaner, and no change to your api/configuration.

I have updated the PR, with the change